include/libzazen.h

```diff
@@ -44,18 +44,51 @@ struct zazen_opengl;
 
 struct zazen_opengl* zazen_opengl_create(struct wl_display* display);
 
+/* zazen_shell */
```

Comment:
[nits]
```suggestion
/* shell */
```
まあこのコメントなくても良い気がするけど

Reply:
ここの表記揺れは `zazen_*`の方でと統一しようかな　

Reply:
僕はコメントほしい。どのobjectは何が外から使われうるのかわかりやすくしたい。